src/apps/curve/positions.test.ts

```diff
@@ -0,0 +1,244 @@
+import { getAllCurvePools } from './positions'
+
+jest.mock('got', () => ({
+  get: jest.fn().mockImplementation((url: string) => {
+    if (url === 'https://api.curve.fi/v1/getPools/celo/factory') {
+      return {
+        json: jest.fn().mockResolvedValue(mockResponse),
+      }
+    }
+    throw new Error('Invalid curve url')
+  }),
+}))
```

Comment:
nit: this repo uses https://mswjs.io/ to mock API requests.

This way mocks are not dependant on a specific network request library.

Mock server is in https://github.com/valora-inc/hooks/blob/main/test/server.ts

Reply:
thanks, sounds good. i refactored to use msw for mocking